client/environment.go

```diff
@@ -165,6 +166,10 @@ type EnvironmentMoveRequest struct {
 	ProjectId string `json:"projectId"`
 }
 
+type EnvironmentDestroyResponse struct {
```

Comment:
The response actually returns the whole deployment log object, but not sure it's required to be explicit here

Reply:
It's fine. Golang will just omit the rest. We can add them in the future if required.
I prefer to keep things to minimum (add fields only when needed).